src/main/java/org/folio/qm/client/SRMChangeManagerClient.java

```diff
@@ -1,21 +1,25 @@
 package org.folio.qm.client;
 
+import org.folio.rest.jaxrs.model.ParsedRecordDto;
 import org.springframework.cloud.openfeign.FeignClient;
 import org.springframework.http.MediaType;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestParam;
 
-import org.folio.rest.jaxrs.model.ParsedRecordDto;
-
 
 @FeignClient(value = "change-manager")
 public interface SRMChangeManagerClient {
 
   @GetMapping(value = "/parsedRecords", produces = MediaType.APPLICATION_JSON_VALUE)
   ParsedRecordDto getParsedRecordByInstanceId(@RequestParam("instanceId") String instanceId);
 
+  @PostMapping(value = "/parsedRecords")
+  void postParsedRecord(@RequestBody  ParsedRecordDto recordDto);
```

Comment:
This method isn't used anywhere. Should it be deleted in this case?

Reply:
removed